### PR TITLE
luyten 0.4.4 (new formula)

### PR DESCRIPTION
**Pull request by makigumo**

### CODE REVIEW

File: Library/Formula/luyten.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  head do
+    url "https://github.com/deathmarine/Luyten.git"
+    depends_on "maven" => :build
+  end
+
+  depends_on :java => "1.7+"
+
+  def install
+    if build.head?
+      version = `mvn org.apache.maven.plugins:maven-help-plugin:evaluate -Dexpression=project.version | grep -v '\\['`.chomp
```
Comment (makigumo) [inline Library/Formula/luyten.rb]: Hmm, why is this executed for the stable build and not only for the HEAD build?

File: Library/Formula/luyten.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  head do
+    url "https://github.com/deathmarine/Luyten.git"
+    depends_on "maven" => :build
+  end
+
+  depends_on :java => "1.7+"
+
+  def install
+    if build.head?
+      ver = `mvn org.apache.maven.plugins:maven-help-plugin:evaluate -Dexpression=project.version | grep -v '\\['`.chomp
```
Comment (mikemcquaid) [inline Library/Formula/luyten.rb]: Use Ruby rather than `grep` to find this, thanks.

Comment (makigumo) [inline Library/Formula/luyten.rb]: Do you mean like this?
```ruby
ver = `mvn org.apache.maven.plugins:maven-help-plugin:evaluate -Dexpression=project.version`.split.grep(/^\d+\.\d+\.\d+/).uniq.first
```

Comment (mikemcquaid) [inline Library/Formula/luyten.rb]: Yep, that works.

**Comment by makigumo** [comment]
I changed the grepping as suggested. Thanks.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**